dev-docs/bidders/blockthrough.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: bidder
+title: Blockthrough
+description: Prebid BT Bidder Adapter
+biddercode: blockthrough
+gvl_id: 815
+usp_supported: true
+coppa_supported: false
+gpp_sids: usp
+schain_supported: true
+dchain_supported: false
+userId: pubProvidedId, id5Id, criteo, sharedId, identityLink, unifiedId, userId
+media_types: banner
+floors_supported: true
+fpd_supported: true
+pbjs: true
+pbs: false
+multiformat_supported: will-not-bid
+ortb_blocking_supported: false
+sidebarType: 1
```

Comment:
Not sure how much this matters or if they're not included because they're not applicable, but I'm seeing some values from the docs that we haven't included here:
```
aliasCode
tcfeu_supported
safeframes_ok
deals_supported
prebid_member
privacy_sandbox
```

Reply:
Since they aren't applicable I think we can skip them